verification/curator-service/ui/src/components/landing-page/ForgotPasswordForm.tsx

```diff
@@ -0,0 +1,153 @@
+import React from 'react';
+import { useFormik } from 'formik';
+import * as Yup from 'yup';
+
+import { makeStyles, Theme } from '@material-ui/core/styles';
+import TextField from '@material-ui/core/TextField';
+import Button from '@material-ui/core/Button';
+import {
+    Dialog,
+    DialogContent,
+    DialogContentText,
+    DialogTitle,
+} from '@material-ui/core';
+
+const useStyles = makeStyles((theme: Theme) => ({
+    checkboxRoot: {
+        display: 'block',
+    },
+    required: {
+        color: theme.palette.error.main,
+    },
+    inpputField: {
+        display: 'block',
+        width: '240px',
+        marginBottom: '10px',
+    },
+    signInButton: {
+        marginTop: '10px',
+    },
+    link: {
+        fontWeight: 'bold',
+        color: theme.palette.primary.main,
+        cursor: 'pointer',
+    },
+    forgotPassword: {
+        fontWeight: 'normal',
+        color: theme.palette.primary.main,
+        cursor: 'pointer',
+        fontSize: 'small',
+        marginTop: '-8px',
+        display: 'flex',
+        justifyContent: 'flex-end',
+    },
+    title: {
+        margin: '10px 0',
+    },
+    formFlexContainer: {
+        display: 'flex',
+        gap: '80px',
+    },
+}));
+
+interface FormValues {
+    email: string;
+    password: string;
+}
+
+interface SignInFormProps {
+    forgotPasswordScreenOn: boolean;
+    setForgotPasswordScreenOn: (active: boolean) => void;
+}
+
+export default function ChangePasswordForm({
+    forgotPasswordScreenOn, setForgotPasswordScreenOn
+}: SignInFormProps) {
+    const classes = useStyles();
+
+    const validationSchema = Yup.object().shape({
+        email: Yup.string()
+            .email('Invalid email address')
+            .required('This field is required'),
+        confirmEmail: Yup.string().test(
+            'emails-match',
+            'Emails must match',
+            function (value) {
+                return this.parent.email === value;
+            },
+        ),
+        password: Yup.string().required('This field is required'),
+        passwordConfirmation: Yup.string().test(
```

Comment:
You can delete `confirmEmail`, `password` and `confirmPassword` as you are not using them in this form